docs/developer-docs/unsupported-emulators-and-cores.md

```diff
@@ -0,0 +1,442 @@
+- ❓ libretro core: **TGB Dual**
+
+## Game Boy Color
+
+- ❌ Standalone emulator: **Pizza Boy GBC**
+  - No longer being developed. Compatibility issues can cause problems with unlocks.
+- ❓ libretro core: **DoubleCherryGB**
+- ❓ libretro core: **Emux GB**
+- ❓ libretro core: **fixGB**
+- ❓ libretro core: **SameBoy**
+- ❓ libretro core: **TGB Dual**
+
+## Game Boy Advance
+
+- ❌ Standalone emulator: **Pizza Boy GBA**
+  - No longer being developed. Compatibility issues can cause problems with unlocks.
+- ❓ libretro core: **gpSP**
+- ❓ libretro core: **Meteor**
+- ❓ libretro core: **TempGBA**
+
+## GameCube
+
+- ⌛ libretro core: **Dolphin**
+  - Testing
+
+## J2ME
+
+- ❌ _Not supported_ - needs console ID, hashing method and memory map
+- ❌ libretro core: **freej2me**
+
+## Magnavox Odyssey 2
+
+- ❓ BizHawk core: **O2Hawk**
+
+## Master System / Mark III
+
+- ❓ BizHawk core: **SMSHawk**
+- ❓ libretro core: **Emux SMS**
+
+## MicroW8
+
+- ❌ _Not supported_
+- ❌ libretro core: **MicroW8**
+
+## MS-DOS
+
+- ❌ _Not supported_ - needs hashing method and memory map
+- ❌ libretro core: **dosbox-core**
+- ❌ libretro core: **dosbox-SVN**
+- ❌ libretro core: **dosbox-pure**
+  - Needs a way to prevent launching with user-specified command line parameters
+  - Needs a way to prevent TSRs
+  - Needs a way to deactivate achievements if game drops to command prompt
+
+## MSX
+
+- ❓ libretro core: **fMSX**
+
+## Neo Geo Pocket
+
+- ❓ libretro core: **RACE**
+
+## Neo Geo Pocket Color
+
+- ❓ libretro core: **RACE**
+
+## NES / Famicom
+
+- ❌ libretro core: **NEStopia**
+  - Does not map SRAM
+- ❓ libretro core: **Emux NES**
+- ❓ libretro core: **fixNES**
+
+## Nintendo 3DS
+
+- ❌ _Not supported_ - needs hashing method and memory map
+- ❌ libretro core: **Citra**
+  - Only exposes 64MB of memory - documentation suggests there should be 128MB.
+  - Does not support save states
+- ❌ libretro core: **Citra 2018**
+- ❌ libretro core: **Citra Canary**
+
+## Nintendo 64
+
+- ⌛ BizHawk core: **Mupen64Plus**
+  - Testing (there are some graphical issues that can be fixed using the angrylion plugin)
+
+## Nintendo DSi
+
+- ❌ libretro core: **DeSmuME**
+- ❌ libretro core: **melonDS**
+
+## Nokia N-Gage
+
+- ❌ _Not supported_ - needs hashing method and memory map
+
+## Oric
+
+- ❌ _Not supported_
+
+## Palm OS
+
+- ❌ _Not supported_
+- ❌ libretro core: **Mu**
+
+## PC Engine CD / TurboGrafx-CD
+
+- ❌ libretro core: **Beetle PCE**
+  - Does not expose RAM needed for PC Engine CD
+
+## PC-6001
+
+- ❌ _Not supported_ - needs hashing method and memory map
+
+## PC-8800
+
+- ❌ libretro core: **quasi88**
+  - NOT RECOMMENDED
+  - PC-88VA not supported
+  - 16 Apr 2022 - Cannot load m3u. Hash fails because core locks disk file.
+  - 16 Apr 2022 - Attempting to load a single disk game in RALibretro just goes to "How many disks?" prompt. Seems to work in RetroArch
+  - 16 Apr 2022 - RALibretro does not provide subsystem interface for loading multi-disk games
```

Comment:
This will get enabled with the next RAlibretro release. Some information here: https://discord.com/channels/310192285306454017/1124566547797655692/1285959561051766918

I'm fine if you want to leave this here for now.

Reply:
I think we should leave it here for now, and then update both pages simultaneously with the next RALibretro release.